store/tikv/async_commit_test.go

```diff
@@ -24,13 +24,19 @@ import (
 	"github.com/pingcap/errors"
 	"github.com/pingcap/kvproto/pkg/kvrpcpb"
 	"github.com/pingcap/tidb/config"
+	"github.com/pingcap/tidb/kv"
 	"github.com/pingcap/tidb/store/mockstore/cluster"
 	"github.com/pingcap/tidb/store/mockstore/unistore"
 	"github.com/pingcap/tidb/store/tikv/tikvrpc"
 )
 
+// testAsyncCommitCommon is used to put utility functions that will be both used by
+// testAsyncCommitSuite and testAsyncCommitFailSuite.
+type testAsyncCommitCommon struct{}
```

Comment:
`testAsyncCommitCommon` functions typically use a `tikvStore`. How about putting the store in `testAsyncCommitCommon`?

Reply:
Ok. How about merge https://github.com/pingcap/tidb/pull/20178 first which is simpler, and I update it then?

Reply:
ok